### DISCUSSION TITLE
[Hellfire Ramparts] Change Chest Unlock

### DISCUSSION THREAD

**Discussion by hn1206** [chromiecraft/chromiecraft#4596] [Suggestions]
Hi everyone.

The last boss landing has been a bug on all servers I have played on and that speaks volume about how hard it must be to fix it.

But until such a fix is made - could we make it so that the chest with the loot is unlocked when killing the "second boss" which also is the one that counts towards completing the dungeon? 

This would make it so that we at least get the loot when the boss bugs out and does not land.

**Comment by Annamaria-CC** [comment]
Bug reports should be made here: https://github.com/chromiecraft/chromiecraft

I did not see a bug report on not landing of this specific boss. If it happens, feel free to make a bug report for it. We will then fix this issue.
